examples/gke_shared_vpc/README.md

```diff
@@ -0,0 +1,20 @@
+# Simple project with GKE shared VPC
+
+This example illustrates how to create a project with a shared VPC from a host project that is GKE suitable.
+
+As shown in this exmaple, GKE shared VPC is only enabled if the "container.googleapis.com" API is in the "activate_apis" variable list.
+
+It will do the following:
+- Create a project
+- Give appropriate iam permissions to the API and GKE service accounts on the host vpc project 
+
+Expected variables:
+- `gsuite_admin_user`
```

Comment:
Do we need gsuite_admin_user for this example? I'd prefer to not require gsuite unless necessary.

Reply:
nope! we don't need it, I just had in there as an error pops up if the gsuite_override.tf file isn't deleted.